dali/base/dasess.cpp

```diff
@@ -1420,7 +1420,7 @@ class CCovenSessionManager: public CSessionManagerBase, implements ISessionManag
         }
 #endif
         if ((ldapconn->getLDAPflags()&(DLF_SAFE|DLF_ENABLED))!=(DLF_SAFE|DLF_ENABLED))
-            return ldapconn->getPermissions(key,obj,udesc,flags);
+            return (SecAccessFlags)ldapconn->getPermissions(key,obj,udesc,flags);
```

Comment:
is ldapconn->getPermissions() something we control? should it return SecAccessFlags?